x/mint/keeper/hooks_test.go

```diff
@@ -393,14 +394,17 @@ func (suite *KeeperTestSuite) TestAfterEpochEnd() {
 			oldSupply := app.BankKeeper.GetSupply(ctx, sdk.DefaultBondDenom).Amount
 			suite.Require().Equal(sdk.NewInt(keeper.DeveloperVestingAmount), oldSupply)
 
-			if tc.expectedError {
-				suite.Require().Error(mintKeeper.AfterEpochEnd(ctx, defaultEpochIdentifier, tc.hookArgEpochNum))
+			err := mintKeeper.AfterEpochEnd(ctx, defaultEpochIdentifier, tc.hookArgEpochNum)
```

Comment:
nit: I think missing a tab here

Reply:
Maybe not sir, I updated the code with a new line but it don't change any logic